modules/apps/commerce/commerce-catalog-web/src/main/java/com/liferay/commerce/catalog/web/internal/portlet/action/UploadCommerceMediaDefaultImageMVCActionCommand.java

```diff
@@ -42,14 +77,183 @@ protected void doProcessAction(
 			_attachmentsUploadResponseHandler, actionRequest, actionResponse);
 	}
 
+	private static final String _PARAMETER_NAME = "imageSelectorFileName";
```

Comment:
Hey @regiCesar21, move these constants to the proper inner class. Please apply this to other pull requests as well. Thx